.github/workflows/syncWithUpstream.yml

```diff
@@ -0,0 +1,29 @@
+name: Sync to Upstream
```

Comment:
I don't understand the need for this workflow, could you elaborate?

Reply:
Sorry I was using this to sync my folks